hack/update-vendor.sh

```diff
@@ -23,4 +23,13 @@ source "${KUBE_ROOT}/hack/ensure-go.sh"
 
 go mod tidy
 go mod vendor
+
+# Copy full dependencies if needed.
+for dep in $(cat ${KUBE_ROOT}/go.vendor); do
+    src=$(go mod download -json ${dep} | jq -r .Dir)
+    dst="${KUBE_ROOT}/vendor/${dep}"
+    cp -af "${src}/" "${dst}"
+    chmod -R +w ${dst}
```

Comment:
This is to make `rm` and the following `cp -af` work properly, right?

Reply:
If we can rely on coreutils being installed as a base, then we could leverage `install` instead cp/chmod

Reply:
This is because the cached go mod folders are all read-only, this restores the permissions you'll find anywhere else in `vendor/`

Reply:
Right, but assuming it is available, `install` can be used to copy and modify the mode in a single step.

Reply:
nm, `install` doesn't handle recursive subdirectories.

Reply:
Do we need coreutils `install` on Mac? If so, I'd like to stick with the above given that more users might be able to use it